_mental_models/saying-no.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: saying-no
+name: Saying "No"
+benefit: Better Decisions
+summary: Saying "yes" to requests because you don't want to upset people can cause problems.
+prerequisites: Confidence, certainty
+---
+
+#### Description
+
+I say "yes", because I don't want to be rude to people and mostly when I say "no", I feel that person feels bad about it. There might be different cases for saying "yes", but in my case, I was thinking that saying yes to an offer, question or a request makes the thing easier.
+
+But saying some requests "yes" can cause some problems, like burn-out or feeling bad. In some cases you might be ended up with saying "yes" to a case where can't handle.
+
+So being able to say "no" to certain things can help to maintain your mental health and can provide better decision making in long-run.
+
+#### Practice
+
+1. Make sure you are relaxed enough and you are not distracting by some emotions like anger.
+2. Find some cases from your previous experiences that you ended up saying "yes" and write down the cases what could have been if you say "no" to these questions. (The questions that caused some problems, frustration or that you felt bad)
```

Comment:
> Find some cases from your previous experiences that you ended up saying "yes" and write down the cases what could have been if you say "no" to these questions. (The questions that caused some problems, frustration or that you felt bad)

Find some cases from your previous experiences that you said "yes" to and write down the cases what could have been if you said "no".

Since this is a training, it needs to be easier. Some people will find it hard to always come up with examples where they always felt something bad. But they can probably come up with more examples where they didn't feel anything and still train their mind in saying no.

Reply:
Also you can divide 2 into multiple parts.

2. Find a case where you said "yes"
3. Write down 3 different scenarios what could have been if you said "no"